source/blog/2014-06-27-spread-to-the-four-corners-of-the-dev.html.markdown

```diff
@@ -115,7 +115,7 @@ I went to the [June Front End London meetup](http://lanyrd.com/2014/front-end-lo
 >
 > Visualise a bunch of open data on an interactive map:
 >
-> ![image](https://dl.dropboxusercontent.com/u/2217931/dev-newsletter-wearedata.png)
```

Comment:
And again - I wrote this post, but Tom wrote the sub post within it that contains this image.